02_programming_fundamentals/08_this_day_1/05_phone_production/__tests__/phone_production.test.js

```diff
@@ -0,0 +1,52 @@
+const fs = require("fs");
+const path = require("path");
+const readcode = require("../../../.test_utils/readcode");
+const {phone, massung, jPhone, personalJPhone, workJPhone} = require("../05_phone_production.js")
+
+let studentCode;
+beforeAll(() => {
+  // Loads the student's code
+  studentCode = readcode(path.resolve(__dirname, "../05_phone_production.js"));
+  return studentCode;
+});
+
+test("Testing if massung has the right properties", () => {
+  massung.functional = true;
+  massung.autodestruction();
+  expect(massung).toHaveProperty('autodestruction');
```

Comment:
It's a detail but for those who are trying to go step by step, the order of the verifications can feel weird.
You're testing first `autodestruction` but that's the last thing you asked for in the exercise.

It's something that I would not advise in a real program but for these tests, could you make a test per property? That way, students see progress :)